modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,31 @@
 = What’s New in Studio
+
+Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
+
+This version also improves the metadata edition and Design Center import experiences and adds a feature to comment and uncomment objects from the canvas.
+
+== Comprehensive Publication to Exchange
+
+When publishing an asset to Exchange, you can now add more information such as tags, categories, name, description, and documentation to your asset directly from Studio.
```

Comment:
```suggestion
When publishing an asset to Exchange, you can now add more information (such as tags, categories, names, descriptions, and documentation) to your assets directly from Studio.
```
I added the parens just to lessen the distraction between "add information" and "to your assets."